src/schema/v2/image/ScaledImage.ts

```diff
@@ -0,0 +1,28 @@
+import {
+  GraphQLString,
+  GraphQLObjectType,
+  GraphQLInt,
+  GraphQLFloat,
+  GraphQLNonNull,
+} from "graphql"
+import { scale } from "proportional-scale"
+import { ResolverContext } from "types/graphql"
+
+export type TScaledImage = ReturnType<typeof scale> & {
+  src: string
+  srcSet: string
+}
+
+export const ScaledImage = new GraphQLObjectType<TScaledImage, ResolverContext>(
+  {
+    name: "ScaledImage",
+    fields: {
+      scale: { type: new GraphQLNonNull(GraphQLFloat) },
+      width: { type: new GraphQLNonNull(GraphQLInt) },
+      height: { type: new GraphQLNonNull(GraphQLInt) },
+      paddingBottom: { type: new GraphQLNonNull(GraphQLString) },
+      src: { type: new GraphQLNonNull(GraphQLString) },
+      srcSet: { type: new GraphQLNonNull(GraphQLString) },
```

Comment:
Just created a new, more sensible version of the resized image type with non-nullable properties and dropping the `url` name.